packages/astro/src/default/components/Header.astro

```diff
@@ -5,6 +5,12 @@ import { LoginButton } from './LoginButton';
 import { ThemeSwitch } from './ThemeSwitch';
 import { useAuth } from './setup';
 
+interface Props {
+  logoLink: string;
```

Comment:
Given it's optional in the schema, we should probably also make it optional here? And not render a link at all of the option is not set.

Reply:
Or default to `href="/"`?

Reply:
It's defaulted to `/` if not set in [`[...slug].astro`](https://github.com/stackblitz/tutorialkit/pull/68/files#diff-9402baf5c6f38adc001724a8d4876230199fa0ff082bb036e839a60ad3195f65R29)